amr-wind/utilities/io_utils.H

```diff
@@ -22,9 +22,9 @@ inline std::string timestamp()
 {
     auto etime = std::chrono::system_clock::now();
     auto etimet = std::chrono::system_clock::to_time_t(etime);
-    char buf[64];
-    ctime_r(&etimet, buf);
-    std::string tstamp(buf);
+    amrex::Array<char, 64> buf;
```

Comment:
Same comment from above, is an amrex array the right type for this?

Reply:
see above